src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2377,118 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("cleva")
+def get_cleva_spec(task: str, version: str, subtask: str = None, prompt_id: int = 0) -> RunSpec:
+    from .scenarios.cleva_scenario import CLEVAScenario  # noqa
+
+    CLEVAScenario.download_dataset()
+
+    prompt_template, prompt_setting = CLEVAScenario.get_prompt_setting(task, subtask, version, prompt_id)
+    inference_parameters = CLEVAScenario.load_inference_parameters(task, subtask, version, prompt_id)
+
+    class_name_prefix = "".join([word.capitalize() for word in task.split("_")])
+    scenario_spec = ScenarioSpec(
+        class_name=f"helm.benchmark.scenarios.cleva_scenario.CLEVA{class_name_prefix}Scenario",
+        args={"version": version, "subtask": subtask, "prompt_template": prompt_template},
```

Comment:
Could you make the arg `"prompt_id": prompt_id` instead of `"prompt_template": prompt_template`, and then fetch the template in `CLEVAScenario.__init__()`?

`ScenarioSpec.args` can't be too long because we use it to generate a file name, and `prompt_template` can be very long.